### PR TITLE
{math}[foss/2022a] NTPoly v2.7.1, ELSI v2.9.1, libmbd v0.12.6 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by seb45tian** [comment]
Test report by @seb45tian
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
r10n47.palma.wwu - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/8e225626171bb6cecdd036e450a435e3 for a full test report.

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/e/ELSI/ELSI-2.9.1-foss-2022a-PEXSI.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ('Bison', '3.8.2'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('ELPA', '2021.11.001'),
+    ('NTPoly', '2.7.1'),
+]
+
+abs_path_compilers = True
```
Comment (smoors) [inline easybuild/easyconfigs/e/ELSI/ELSI-2.9.1-foss-2022a-PEXSI.eb:38]: add comment why this is needed?

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @lexming!

**Pull request merged**